apiserver/facades/agent/secretsmanager/secrets.go

```diff
@@ -533,6 +534,8 @@ func (s *SecretsManagerAPI) getRemoteSecretContent(uri *coresecrets.URI, refresh
 	if err != nil {
 		return nil, nil, false, errors.Annotate(err, "creating remote secret client")
 	}
+	defer func() { _ = extClient.Close() }()
```

Comment:
This is the fix.

Reply:
Do we want to do some form of logging an error that isn't nil instead?